src/main/java/org/molgenis/vcf/report/AppCommandLineOptions.java

```diff
@@ -74,13 +78,26 @@
             .desc(
                 "Comma-separated list of sample-phenotypes (e.g. HPO:123 or HPO:123;HPO:234 or sample0/HPO:123,sample1/HPO:234). Phenotypes are CURIE formatted (prefix:reference) and separated by a semicolon.")
             .build());
+    appOptions.addOption(
+        Option.builder(OPT_MAX_RECORDS)
+            .hasArg(true)
+            .longOpt(OPT_MAX_RECORDS_LONG)
+            .desc(
+                "Integer stating the maximum number of records to be available in the report.")
```

Comment:
suggestion: add default (ReportGeneratorSettings.DEFAULT_MAX_NR_RECORDS) to the description.